GoogleForCreators/web-stories-wp
# Adding Google Sans font to dashboard

Adding Google Sans font for use in dashboard. Part of initial set up work for Dashboard, reference #547. Includes initial boiler plate for styled component globals and theme usage to enable testing of font usability. 

To test:
- see that the h1 on  wp-admin/edit.php?post_type=web-story&page=stories-dashboard is now using Google Sans 


![Screen Shot 2020-03-12 at 11 02 03 AM](https://user-images.githubusercontent.com/10720454/76553916-293a3d80-6452-11ea-8ae1-93b05126b774.png)
